introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+### Node Types
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes
+are not aware of cluster-wide distribution policies, and otherwise look just as if they were
+single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
+from accidentally executing operations on data nodes that might endanger the data consistency
+of distributed hypertables.
+
+It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
+
+### Distributed Databases
+
+Distributed databases represent the highest level in the hierarchy used to define a multi-node
+TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
+that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
+different nodes play both the role of access node and data node for different databases.
```

Comment:
I'm not sure this is correct, and I'm a bit troubled by the terminology here.  We currently disallow a access node from being added as a data node in a different distributed database.  We also disallow adding an instance which is already a data node as a data node for a different distributed database.

What we do allow is having different timescale instances on a single postgres cluster (running in the same postgres server process), and these can form distributed databases which are wholly contained in that postgres cluster.  However, trying to relate TimescaleDB Clusters to PostgresSQL clusters is problematic, so I'm not really sure how this should be phrased.

Reply:
Gotcha, I completely changed this section and that makes total sense. I forgot that we had added in that disallowance

Reply:
New paragraph: 

### Distributed Databases

A multi-node TimescaleDB cluster contains distributed databases. Distributed databases should be configured 
to have one primary access node and multiple data nodes. An access node cannot also be a data node. 
We currently only support a single primary access node that routes all write queries.

Reply:
@dianasaur323 new paragraph above captures it well. Better to explain the supported configuration directly.

 In terms of terminology, overloading the word "cluster" is bound to lead to confusion. But it's actually pretty clear from the context. I think it reads well but we should be prepared to disambiguate Postgres/TimescaleDB clusters.

my only question re: "databases" is the pluralization - are we planning to support multiple databases on a cluster, all sharing one access node? Given the restriction that an access node cannot double as a data node, is there any use case for having more than one database per cluster?

Reply:
Awesome, I'm glad it's a little clearer. My understanding is that you might want more than one database per cluster for something like permissioning if you want to enforce different roles? Although I'm not the most equipped to know exactly why someone would want to do this.

Reply:
Hmmm I'm not entirely sure exactly how to address the pluralization issue, so I'm leaning towards shipping this as is and seeing what people do with it. haha!